docs/uninstall/uninstall_everest_helm.md

```diff
@@ -189,51 +189,95 @@ EOF
+    "vmalertmanagerconfigs.operator.victoriametrics.com" deleted
+    customresourcedefinition.apiextensions.k8s.io 
+    "vmalertmanagers.operator.victoriametrics.com" deleted
+    customresourcedefinition.apiextensions.k8s.io 
+    "vmalerts.operator.victoriametrics.com" deleted
+    customresourcedefinition.apiextensions.k8s.io 
+    "vmagents.operator.victoriametrics.com" deleted
+    customresourcedefinition.apiextensions.k8s.io 
+    "vmclusters.operator.victoriametrics.com" deleted
+    customresourcedefinition.apiextensions.k8s.io 
+    "vmusers.operator.victoriametrics.com" deleted
+    customresourcedefinition.apiextensions.k8s.io 
+    "vmscrapeconfigs.operator.victoriametrics.com" deleted
+    customresourcedefinition.apiextensions.k8s.io 
+    "vlogs.operator.victoriametrics.com" deleted
+    customresourcedefinition.apiextensions.k8s.io 
+    "databaseclusterbackups.everest.percona.com" deleted
+    customresourcedefinition.apiextensions.k8s.io 
+    "databaseclusters.everest.percona.com" deleted
+    customresourcedefinition.apiextensions.k8s.io 
+    "databaseengines.everest.percona.com" deleted
+    customresourcedefinition.apiextensions.k8s.io 
+    "backupstorages.everest.percona.com" deleted
+    customresourcedefinition.apiextensions.k8s.io   
+    "databaseclusterrestores.everest.percona.com" deleted
+    customresourcedefinition.apiextensions.k8s.io 
+    "monitoringconfigs.everest.percona.com" deleted
+    customresourcedefinition.apiextensions.k8s.io 
+    "perconaxtradbclusterbackups.pxc.percona.com" deleted
+    customresourcedefinition.apiextensions.k8s.io 
+    "perconaxtradbclusterrestores.pxc.percona.com" deleted
+    customresourcedefinition.apiextensions.k8s.io 
+    "perconaxtradbclusters.pxc.percona.com" deleted
+    customresourcedefinition.apiextensions.k8s.io 
+    "perconaservermongodbbackups.psmdb.percona.com" deleted
+    customresourcedefinition.apiextensions.k8s.io 
+    "perconaservermongodbrestores.psmdb.percona.com" deleted
+    customresourcedefinition.apiextensions.k8s.io 
+    "perconaservermongodbs.psmdb.percona.com" deleted
+    customresourcedefinition.apiextensions.k8s.io 
+    "perconapgbackups.pgv2.percona.com" deleted
+    customresourcedefinition.apiextensions.k8s.io 
+    "perconapgclusters.pgv2.percona.com" deleted
+    customresourcedefinition.apiextensions.k8s.io 
+    "perconapgrestores.pgv2.percona.com" deleted
+    customresourcedefinition.apiextensions.k8s.io 
+    "perconapgupgrades.pgv2.percona.com" deleted
+    customresourcedefinition.apiextensions.k8s.io 
+    "crunchybridgeclusters.postgres-operator.crunchydata.com" deleted
+    customresourcedefinition.apiextensions.k8s.io "pgadmins.postgres- 
+    operator.crunchydata.com" deleted
+    customresourcedefinition.apiextensions.k8s.io "pgupgrades.postgres- 
+    operator.crunchydata.com" deleted
+    customresourcedefinition.apiextensions.k8s.io 
+    "postgresclusters.postgres-operator.crunchydata.com" deleted
```

Comment:
Why did you add line breaks for this output? Now, it doesn't reflect what the user sees in the terminal when running the command...

Reply:
I attempted to remove the copy button, which caused this issue. I have undone the changes.